cmd/admission/app/configure/configure.go

```diff
@@ -60,17 +62,30 @@ func (c *Config) AddFlags() {
 	flag.StringVar(&c.KeyFile, "tls-private-key-file", c.KeyFile, "File containing the default x509 private key matching --tls-cert-file.")
 	flag.StringVar(&c.CaCertFile, "ca-cert-file", c.CaCertFile, "File containing the x509 Certificate for HTTPS.")
 	flag.IntVar(&c.Port, "port", 443, "the port used by admission-controller-server.")
-	flag.StringVar(&c.MutateWebhookConfigName, "mutate-webhook-config-name", "volcano-mutate-job",
-		"Name of the mutatingwebhookconfiguration resource in Kubernetes.")
-	flag.StringVar(&c.MutateWebhookName, "mutate-webhook-name", "mutatejob.volcano.sh",
-		"Name of the webhook entry in the webhook config.")
-	flag.StringVar(&c.ValidateWebhookConfigName, "validate-webhook-config-name", "volcano-validate-job",
-		"Name of the mutatingwebhookconfiguration resource in Kubernetes.")
-	flag.StringVar(&c.ValidateWebhookName, "validate-webhook-name", "validatejob.volcano.sh",
-		"Name of the webhook entry in the webhook config.")
+	flag.StringVar(&c.MutateWebhookConfigName, "mutate-webhook-config-name", "",
```

Comment:
I remember there is a method called `MarkDeprecated` , deprecate explicitly.

Reply:
It's in spf13 package, can it be used here?

Reply:
lol